internal/service/service_test.go

```diff
@@ -0,0 +1,143 @@
+package service_test
+
+import (
+	"bytes"
+	"fmt"
+	"github.com/andreipimenov/golang-training-2021/internal/mock"
+	"github.com/andreipimenov/golang-training-2021/internal/model"
+	"github.com/andreipimenov/golang-training-2021/internal/repository"
+	"github.com/andreipimenov/golang-training-2021/internal/service"
+	"github.com/rs/zerolog"
+	"github.com/stretchr/testify/suite"
+	"io/ioutil"
+	"net/http"
+	"testing"
+	"time"
+)
+
+var (
+	price = model.Price{
+		Open:  "99.9",
+		High:  "99.9",
+		Low:   "99.9",
+		Close: "99.9",
+	}
+	apiKeyValid     = "asdasd"
+	apiKeyInvalid   = ""
+	tickerValid     = "AAPL"
+	tickerInvalid   = "ASDASD"
+	dateValid       = "2021-07-26"
+	dateLayoutValid = "2006-01-02"
+	stockApiFormat  = "https://www.alphavantage.co/query?function=TIME_SERIES_DAILY&outputsize=full&symbol=%s&apikey=%s"
+)
+
+type serviceTestSuite struct {
+	suite.Suite
+}
+
+func TestService(t *testing.T) {
+	suite.Run(t, new(serviceTestSuite))
+}
+
+func (suite *serviceTestSuite) TestHitCache() {
```

Comment:
Perhaps it is worth more explicitly describing the name of the component under test in the test name.
For example TestServiceHitCache() instead of TestHitCache()